core/model/modx/processors/resource/getnodes.class.php

```diff
@@ -128,7 +128,7 @@ public function getRootNode() {
      */
     public function getContextQuery() {
         $this->itemClass= 'modContext';
-        $c= $this->modx->newQuery($this->itemClass, array('key:!=' => 'mgr'));
+        $c= $this->modx->newQuery($this->itemClass, array('key:!=' => 'mgr', 'name:LIKE' => "%{$this->getProperty('contextFilter', '')}%"));
```

Comment:
I think it would be better to separate this from the base query. Having ```LIKE %%``` on every query in no property sent seems not optimal.